src/Compilers/VisualBasic/Portable/Symbols/Source/OverrideHidingHelper.vb

```diff
@@ -574,7 +574,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Inherits OverrideHidingHelper
 
         ' Comparer for comparing signatures of TSymbols in a runtime-equivalent way
-        Private Shared ReadOnly s_runtimeSignatureComparer As IEqualityComparer(Of TSymbol)
+        Private Shared s_runtimeSignatureComparer As IEqualityComparer(Of TSymbol)
```

Comment:
I suspect it is a build vestige of previous changes. I'll try reverting it, though this change must go into master soon (otherwise there is a violation of CLR protection rules in the static constructor below).